### ISSUE TITLE
eddelbuettel.github.io Links 404

### ISSUE THREAD

**Issue by zerweck** [eddelbuettel/drat#108]
Example:
https://eddelbuettel.github.io/drat/DratFAQ.html

**Comment by eddelbuettel** [comment]
That has to do with the new documentation.  I had caught one previous one.  

What was the "originating" link for that, I what page had the broken link you tried to follow?

"Good" documentation should be at https://eddelbuettel.github.io/drat -- I will try my best to fix old links.  The help is appreciated!

**Comment by zerweck** [comment]
I found this link originally on http://eddelbuettel.github.io/drat/ and http://dirk.eddelbuettel.com/code/drat.html 
The second page is the one linked in the URL field of the package DESCRIPTION file

**Comment by zerweck** [comment]
All vignette links have the same 404, btw:
https://eddelbuettel.github.io/drat/DratForPackageAuthors.html
https://eddelbuettel.github.io/drat/DratForPackageUsers.html
https://eddelbuettel.github.io/drat/WhyDrat.html

**Comment by eddelbuettel** [comment]
Let me rephrase this.  You are being most helpful in providing me links that are broken.

Where i.e. on which page are you reading these links?   If you start from

    https://eddelbuettel.github.io/drat/

and follow 'Articles' to 

   https://eddelbuettel.github.io/drat/dratfaq/

you get the FAQ and its links at the 'table of content' on the left margin work.

But that is not where you started from.  Which ... was where?

**Comment by zerweck** [comment]
Ah, I meant from the list of vignettes listed here https://eddelbuettel.github.io/drat/#getting-started but I see now that there are multiple ways to get to the FAQ (To be honest, I would not have looked for a FAQ when clicking on a link named Articles, maybe worth a consideration?)

From http://dirk.eddelbuettel.com/code/drat.html there is the same link under Getting Started and also here: https://github.com/eddelbuettel/drat/blob/master/README.md#L46

**Comment by eddelbuettel** [comment]
> To be honest, I would not have looked for a FAQ when clicking on a link named Articles, maybe worth a consideration?

That just occurred to me too.  I guess with _multiple_ vignettes I should rather have a top-level intro / table of content jumping to all four.

> I meant from the list of vignettes listed here  https://eddelbuettel.github.io/drat/#getting-started 

Great!  That may also affect the overall README (ah yes and you noted that). I will chase that down and fix it.  *Much appreciated.*

**Comment by eddelbuettel** [comment]
Actually -- the better fix is to 'revert' in the new documentation setup to the old filenames for the vignettes. I was foolish to rename them as that is in fact bit rot.  Lemme see if I can host at GitHub Pages using softlinks...

**Comment by eddelbuettel** [comment]
It looks like it didn't work but I just realized I once knew how to create / used for another small html files that contain a redirect. 

(Forgot to save this comment.)

These have now been pushed _and work_ though I can / should maybe lower the timeout.  I owe you for the heads-up.  That was a very bad black eye, entirely self-inflicted (though we do have nicer docs now :grinning:) and it is better now!  

I leave this open til I edit the index page and the README.

**Comment by eddelbuettel** [comment]
Did one more pass and placed vignettes inside `vignettes/`.

**Comment by eddelbuettel** [comment]
This can be closed, the lack of following suggests the issue is addressed.  If more is needed please reopen and ideally demonstrate with small reproducible examples why.
